.tekton/central-db-pull-request.yaml

```diff
@@ -240,10 +239,39 @@ spec:
       - name: source
         workspace: workspace
 
+    - name: determine-image-tag
+      params:
+        - name: image
+          value: $(params.output-image)
+      runAfter:
+      - clone-repository
+      taskSpec:
+        params:
+          - name: image
+            description: Image without tag
+        results:
+          - name: image-with-tag
+            description: Fully Qualified Image Name with identifying tag
+        steps:
+          - name: determine-image-tag
+            image: quay.io/stackrox-io/apollo-ci:stackrox-build-0.3.66
+            env:
+              - name: IMAGE
+                value: $(params.image)
+            script: |
+              #!/usr/bin/env bash
+              set -euo pipefail
+              cd "$(workspaces.source.path)/source"
+              echo -n "${IMAGE}:$(make tag)" > "$(results.image-with-tag.path)"
+              cat "$(results.image-with-tag.path)"
```

Comment:
Suggesting `echo ... | tee blah` instead of `echo ... > blah; cat blah` if the idea is to show the contents.